content/docs/glossary/sponsored-reserves.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Sponsored Reserves
+order:
+---
+
+Protocol 14 introduces operations that allow an account to pay the base reserves for another account. This is done by establishing a is-sponsoring-future-reserves-for relationship using the [Begin Sponsoring Future Reserves](../start/list-of-operations.mdx#begin-sponsoring-future-reserves) operation, where the sponsoring account is the source account, and is terminated by [End Sponsoring Future Reserves](../start/list-of-operations.mdx#end-sponsoring-future-reserves), where the sponsored account is the source account. Both operations must appear in a single transaction, which guarantees that both the sponsoring and sponsored accounts agree to every sponsorship.
```

Comment:
The second sentence "This is done by..." is a pretty tough read, in my opinion. Might help to split that into two simpler sentences.

Reply:
I updated that sentence.